dmsgpty/conf.go

```diff
@@ -41,7 +41,10 @@ func WriteConfig(conf Config, path string) error {
 	}
 	enc := json.NewEncoder(f)
 	enc.SetIndent("", "    ")
-	return enc.Encode(&conf)
+	if err = enc.Encode(&conf); err != nil {
+		return err
+	}
+	return f.Close()
```

Comment:
Should we not run this as a defered call?

Reply:
Windows will complain about it, since the file descriptor theoretically can still be in used and not closed before using it again, already tested it.